src/utils/registration-helpers.ts

```diff
@@ -258,27 +258,27 @@ export const contributorConfig: ContributorConfig = {
   },
   // Book
   [BookType.AcademicMonograph]: {
-    primaryRoles: [ContributorRole.Creator],
+    primaryRoles: [ContributorRole.Creator, ContributorRole.Editor],
     secondaryRoles: [ContributorRole.ContactPerson, ContributorRole.RightsHolder, ContributorRole.Other],
   },
   [BookType.NonFictionMonograph]: {
-    primaryRoles: [ContributorRole.Creator],
+    primaryRoles: [ContributorRole.Creator, ContributorRole.Editor],
     secondaryRoles: [ContributorRole.ContactPerson, ContributorRole.RightsHolder, ContributorRole.Other],
   },
   [BookType.PopularScienceMonograph]: {
-    primaryRoles: [ContributorRole.Creator],
+    primaryRoles: [ContributorRole.Creator, ContributorRole.Editor],
     secondaryRoles: [ContributorRole.ContactPerson, ContributorRole.RightsHolder, ContributorRole.Other],
   },
   [BookType.Textbook]: {
-    primaryRoles: [ContributorRole.Creator],
+    primaryRoles: [ContributorRole.Creator, ContributorRole.Editor],
     secondaryRoles: [ContributorRole.ContactPerson, ContributorRole.RightsHolder, ContributorRole.Other],
   },
   [BookType.Encyclopedia]: {
-    primaryRoles: [ContributorRole.Creator],
+    primaryRoles: [ContributorRole.Creator, ContributorRole.Editor],
     secondaryRoles: [ContributorRole.ContactPerson, ContributorRole.RightsHolder, ContributorRole.Other],
   },
   [BookType.ExhibitionCatalog]: {
-    primaryRoles: [ContributorRole.Creator],
+    primaryRoles: [ContributorRole.Creator, ContributorRole.Editor],
```

Comment:
Har spurt om redaktør også skal støttes for `ExhibitionCatalogChapter`, eller om det bare skal være for `ExhibitionCatalog`. Venter på svar